packages/cycle-scripts/template/config/javascript/flavor.js

```diff
@@ -0,0 +1,24 @@
+const dependencies = []
```

Comment:
1. thought it is cleaner to seperate it, dont mind really
2. My reason is that it is not part of the template source and should not be copied over, we could however remove the structure, move the files one directory up and rename them to `<language>.js`